src/main/resources/application.properties

```
@@ -1,10 +1,9 @@
-spring.datasource.url=jdbc:mysql://localhost:3306/book_shop
-spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
-spring.datasource.username=root
-spring.datasource.password=1111
-changeLogFile=classpath:/db/changelog/db.changelog-master.yaml
-spring.jpa.hibernate.ddl-auto=validate
-spring.jpa.show-sql=true
-
-jwt.expiration=30000000000
-jwt.secret=my_secret_key68937y3hjhsdfghjytdgdbdyyunjhunynyk8uu35465768uhgf
+##spring.datasource.url=jdbc:mysql://mysql-db:3306/book_shop
+#spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
+##spring.datasource.username=root
+##spring.datasource.password=1111
+#changeLogFile=classpath:/db/changelog/db.changelog-master.yaml
+#spring.jpa.hibernate.ddl-auto=validate
+#spring.jpa.show-sql=true
+#jwt.expiration=30000000000
+#jwt.secret=my_secret_key68937y3hjhsdfghjytdgdbdyyunjhunynyk8uu35465768uhgf
```

Why do you need this? remove or uncomment
```suggestion
##spring.datasource.url=jdbc:mysql://mysql-db:3306/book_shop
#spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
##spring.datasource.username=root
##spring.datasource.password=1111
#changeLogFile=classpath:/db/changelog/db.changelog-master.yaml
#spring.jpa.hibernate.ddl-auto=validate
#spring.jpa.show-sql=true
#jwt.expiration=30000000000
#jwt.secret=my_secret_key68937y3hjhsdfghjytdgdbdyyunjhunynyk8uu35465768uhgf
```